### ISSUE TITLE
No image mean subtraction in inception v3?

### ISSUE THREAD

**Issue by kalkaneus** [tensorflow/models#957]
Why there is no subtraction of image with its mean (like subtracting with [123, 117, 104] or with tf.reduce_mean) in inception model? Or is it just me who miss the code?

**Comment by drpngx** [comment]
This is probably too simple to warrant a new function. We generally use `tf.nn.batch_normalization` to normalize across a batch.
